### PR TITLE
fix for rails 5

### PR DESCRIPTION
**Pull request by kevinhughes27**
In Rails 5 controller parameters are no longer a hash and are instead an instance of `ActionController::Parameters` causing the `teaspoon-bundle` to crash in the view. 

Easiest fix is to convert it back into a hash using http://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-to_h. 

`to_h` can be safely called on regular ruby hashes as well having no affect.

I added a new app under the test directory to show this working.

@bouk

### CODE REVIEW

File: test/demo/README.md [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
```
Comment (kevinhughes27) [inline test/demo/README.md:24]: I also removed the readme from the other demo app.

**Comment by bouk** [comment]
I'd be fine with just converting demo to rails 5 too, no need to duplicate everything.

**Comment by kevinhughes27** [comment]
I didn't want to override it since some big applications are still on 4.2.6 I thought we should keep it around.

**Comment by bouk** [comment]
please add the `demo_rails5` directory to `bin/test` then (just copy and change the last 3 lines)

**Comment by kevinhughes27** [comment]
I added to `bin/test` I also added cases that use the `files` syntax

File: bin/test [hunk 2]
```diff
@@ -3,3 +3,15 @@ set -e
...
+
+pushd test/demo>/dev/null
+bundle exec rake teaspoon files=spec/javascripts/whatever_spec.js
+popd>/dev/null
+
+pushd test/demo_rails5>/dev/null
+bundle exec rake teaspoon
+popd>/dev/null
+
+pushd test/demo_rails5>/dev/null
```
Comment (bouk) [inline bin/test:22]: you can remote these intermediary pushd and popd

**Comment by kevinhughes27** [comment]
Success:
image

**Comment by bouk** [comment]
thank you, will cut a release

**Pull request merged**